amlip_py/_setup.py

```diff
@@ -0,0 +1,39 @@
+"""Setup file to create amlip_py library."""
+from setuptools import setup
+
+package_name = 'amlip_py'
+
+description = 'AML-IP AML Tool'
+long_description = description
+# with open('README.md', 'r') as f:
+#     long_description = f.read()
+
+file_packages = [
+    package_name,
+    package_name + '/node',
+    package_name + '/types',
+]
+
+# TODO remove
+variable = 'this variable is veeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeery long for python'
```

Comment:
should fail?

Reply:
not anymore. Lenght allowed is up to 99.